defs/support.json

```diff
@@ -1,100 +1,245 @@
-		"Bcash": "2.0.7",
-		"Bcash Testnet": "2.0.7",
-		"Bgold": "2.0.7",
-		"Bprivate": "2.0.7",
-		"Dash": "2.0.5",
-		"DigiByte": "2.0.5",
-		"Dogecoin": "2.0.5",
-		"Fujicoin": "2.0.5",
-		"Groestlcoin": "2.0.8",
-		"Groestlcoin Testnet": "2.0.8",
-		"Litecoin": "2.0.5",
-		"Litecoin Testnet": "2.0.7",
-		"Monacoin": "2.0.5",
-		"Namecoin": "2.0.5",
-		"Vertcoin": "2.0.5",
-		"Viacoin": "2.0.7",
-		"Zcash": "2.0.7",
-		"Zcash Testnet": "2.0.7",
-		"Zcoin": "2.0.7",
-		"Zcoin Testnet": "2.0.7",
-		"Zencash": "2.0.8"
-	},
-	"connect": {
-		"Bitcoin": true,
-		"Testnet": true,
-		"Bcash": true,
-		"Bgold": true,
-		"Dash": true,
-		"Litecoin": true,
-		"Zcash": true
-	},
-	"webwallet": {
-		"Bitcoin": true,
-		"Testnet": true,
-		"Bcash": true,
-		"Bgold": true,
-		"Bprivate": true,
-		"Dash": true,
-		"Decred": true,
-		"Decred Testnet": true,
-		"DigiByte": true,
-		"Dogecoin": true,
-		"Fujicoin": true,
-		"Groestlcoin": true,
-		"Groestlcoin Testnet": true,
-		"Litecoin": true,
-		"Litecoin Testnet": true,
-		"Monacoin": true,
-		"Vertcoin": true,
-		"Viacoin": true,
-		"Zcash": true,
-		"Zcash Testnet": true,
-		"Zcoin": true
-	},
-	"electrum": {
-		"Bitcoin": "https://electrum.org",
-		"Bcash": "https://electroncash.org",
-		"Bgold": "https://github.com/BTCGPU/electrum",
-		"Bprivate": "https://github.com/BTCPrivate/electrum-btcp",
-		"Dash": "https://electrum.dash.org",
-		"Groestlcoin": "https://www.groestlcoin.org/groestlcoin-electrum-wallet/",
-		"Koto": "https://electrum.kotocoin.info",
-		"Litecoin": "https://electrum-ltc.org",
-		"Monacoin": "https://electrum-mona.org",
-		"Namecoin": "https://github.com/namecoin/electrum-nmc",
-		"Viacoin": "https://vialectrum.org",
-		"Zcoin": "https://github.com/zcoinofficial/electrum-xzc"
-	}
+    "btc:BCH": {
+        "trezor1": "1.6.2",
+        "trezor2": "2.0.7",
+        "connect": true,
+        "webwallet": true,
+        "other": {
+            "Electron Cash": "https://electroncash.org"
+        }
+    },
+    "btc:BTC": {
+        "trezor1": "1.5.2",
+        "trezor2": "2.0.5",
+        "connect": true,
+        "webwallet": true,
+        "other": {
+            "Electrum": "https://electrum.org"
+        }
+    },
+    "btc:BTCP": {
+        "trezor1": "1.6.2",
+        "trezor2": "2.0.7",
+        "connect": null,
+        "webwallet": true,
+        "other": {
+            "BTCP Electrum": "https://github.com/BTCPrivate/electrum-btcp"
+        }
+    },
+    "btc:BTG": {
+        "trezor1": "1.6.2",
+        "trezor2": "2.0.7",
+        "connect": true,
+        "webwallet": true,
+        "other": {
+            "ElectrumG": "https://github.com/BTCGPU/electrum"
+        }
+    },
+    "btc:DASH": {
+        "trezor1": "1.5.2",
+        "trezor2": "2.0.5",
+        "connect": true,
+        "webwallet": true,
+        "other": {
+            "Dash Electrum": "https://electrum.dash.org"
+        }
+    },
+    "btc:DCR": {
+        "connect": null,
+        "electrum": null,
```

Comment:
Does it make sense to have `"electrum": null,`, when we put Electrum links in `other` when defined?